apps/src/codebridge/FilePreview/HTMLPreview.tsx

```diff
@@ -134,7 +133,7 @@ export const HTMLPreview = () => {
     <PanelContainer
       id={'html-preview'}
       headerContent={codebridgeI18n.preview()}
-      rightHeaderContent={<RightButtons />}
```

Comment:
I _think_ everything in `RightButtons` is either Pythonlab-specific or will appear in resources bar (eg, version history, settings). Could  one of y'all confirm?

Reply:
Yep, there's also the widget mode toggle but I think that's now covered by the new buttons.